src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/render/ModuleRotations.kt

```diff
@@ -87,7 +90,28 @@ object ModuleRotations : Module("Rotations", Category.RENDER) {
         val server = RotationManager.serverRotation
         val current = RotationManager.currentRotation
 
+        // Apply smoothing if enabled
+        if (smoothRotations && current != null && lastRotation != null) {
+            val smoothedRotation = smoothRotation(lastRotation!!, current)
+            lastRotation = smoothedRotation
+            return smoothedRotation
+        }
+
+        lastRotation = current ?: server
         return current ?: server
     }
 
+    /**
+     * Rotation Smoothing
+     */
+    private fun smoothRotation(from: Rotation, to: Rotation): Rotation {
+        val diffYaw = to.yaw - from.yaw
+        val diffPitch = to.pitch - from.pitch
+        val smoothingFactor = 0.25f
```

Comment:
smoothing factor is hardcoded as it should 👍🏻

Reply:
it should have been a slider inside a toggleableconfigurable but i lost 99.9% of my braincells trying to move all of this shit to a class

Reply:
bruh, just use a ToggleableConfigurable